test/features/page_objects/sdk.rb

```diff
@@ -93,8 +95,9 @@ def cross_device_link
   end
 end
 
-Given(/^I navigate to the SDK$/) do
+Given(/^I navigate to the SDK with "([^"]*)"$/) do |locale|
```

Comment:
same here, you could make it more generic and backwards compatible by changing ` with "([^"]*)"` to `(?:| with (.+)?)`, which gives:
`Given(/^I navigate to the SDK with(?:| with (.+)?)$/)`